plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseTestMySqlTableStatisticsTest.java

```diff
@@ -335,12 +335,12 @@ public void testNumericCornerCases()
 //                            "('mixed_infinities_and_numbers', null, 4.0, 0.0, null, null, null)," +
 //                            "('nans_only', null, 1.0, 0.5, null, null, null)," +
 //                            "('nans_and_numbers', null, 3.0, 0.0, null, null, null)," +
-                            "('large_doubles', null, 1.9, 0.050000000000000044, null, null, null)," +
-                            "('short_decimals_big_fraction', null, 1.9, 0.050000000000000044, null, null, null)," +
-                            "('short_decimals_big_integral', null, 1.9, 0.050000000000000044, null, null, null)," +
-                            "('long_decimals_big_fraction', null, 1.9, 0.050000000000000044, null, null, null)," +
-                            "('long_decimals_middle', null, 1.9, 0.050000000000000044, null, null, null)," +
-                            "('long_decimals_big_integral', null, 1.9, 0.050000000000000044, null, null, null)," +
+                            "('large_doubles', null, 2.0, 0.0, null, null, null)," +
```

Comment:
why does mysql `UNKNOWN_NULL_FRACTION_REPLACEMENT` affects other stats?

Reply:
This is due to `StatsNormalizer` attempting to compensate for the previously logically incorrect estimate of NDV=2, rowCount=2 but also 10% nulls.